src/Tasks/dependency-check-build-task/dependency-check-build-task.ts

```diff
@@ -159,14 +192,14 @@ async function run() {
         // Process scan artifacts is required
         let processArtifacts = !failed || isViolation;
         if (processArtifacts) {
-            console.log('##[debug]Attachments:');
+            logDebug('Attachments:');
             let reports = tl.findMatch(reportsDirectory, '**/*.*');
             reports.forEach(filePath => {
                 let fileName = path.basename(filePath).replace('.', '%2E');
                 let fileExt = path.extname(filePath);
-                console.log(`##[debug]Attachment name: ${fileName}`);
```

Comment:
Changed to debug function call so debug messages will only be printed out if generic pipeline debug variable system.debug=true